tags.go

```diff
@@ -230,34 +239,6 @@ func (client *Client) CreateTag(input TagCreateInput) (*Tag, error) {
 
 //#endregion
 
-//#region Retrieve
-
-// Deprecated: use client.GetServiceWithAlias(alias).Tags instead
```

Comment:
lets make sure we make changelog entries that we are removing these deprecated functions